src/backend/base/langflow/initial_setup/setup.py

```diff
@@ -597,8 +597,8 @@ async def load_flows_from_directory() -> None:
         # Ensure that the default folder exists for this user
         _ = await get_or_create_default_folder(session, user.id)
 
-        async for file_path in anyio.Path(flows_path).iterdir():
```

Comment:
`iterdir` is blocking on Python 3.13. See https://github.com/agronholm/anyio/issues/873

Reply:
Will be fixed in next anyio version: https://github.com/agronholm/anyio/pull/874